docs/tech-reference/plugin_conf.rst

```diff
@@ -88,20 +88,22 @@ the contents of a repository to exactly be the collection of modules installed
 in a puppet environment. This allows you to use Pulp's repository management
 features to manage which modules are installed in puppet.
 
-This distributor starts by deleting every directory it finds in the
-``install_path``, and then it extracts each module in the repository to that
-directory.
+This distributor starts by creating a temporary directory in the parent directory of
```

Comment:
Maybe rewrite these as a numbered list? It would avoid the 'then' ... 'then' ... 'then' language. I'm OK as is also.